### ISSUE TITLE
Spin triggered even clicking outside

### ISSUE THREAD

**Issue by varunsahni003** [Sayegh7/ngx-wheel#3]
Whenever you will click outside this component, even then spinning of wheel is triggered. Suppose there is a page having a paragraph, heading and this ngx-wheel component. On clicking on paragraph or heading also, the wheel will start rotating.

**Comment by Sayegh7** [comment]
@varunsahni003 Hello, I'm not sure I fully understand the issue. Do you have some example code I could use to reproduce the issue?

**Comment by varunsahni003** [comment]
Hi Sayegh
<div>
 <h1>This is heading</h1>
 <p>This is paragraph</p>
 <ngx-wheel>Just example to make to understand issue</ngx-wheel>
<div>

In the above code you can see that there is a component which is having a
heading and a paragraph and your ngx-wheel. When I am clicking on this
paragraph or heading or any other thing (angular component) which is there
on the current opened page, it will spin the wheel. I am not clicking on
the wheel though. The click event is effecting parent elements of wheel.

*In Short*: Click on paragrah tag on page and wheel will spin, click on
heading, wheel spins, if there is navbar on page - click on it and wheel
starts spinning.

*Suggestion*: Apart from this, I want to suggest you that on dragging mouse
on to your ngx-wheel component change the mouse pointer to cursor so that
it looks clickable.

**Comment by Sayegh7** [comment]
@varunsahni003 I see what you mean. Thanks for your feedback 👍. I will release a patch within the next 24 hours.

**Comment by Sayegh7** [comment]
Fixed in 3.0.5
